src/client/Microsoft.Identity.Client/ManagedIdentity/AppServiceManagedIdentitySource.cs

```diff
@@ -19,8 +19,6 @@ internal class AppServiceManagedIdentitySource : ManagedIdentitySource
         private const string AppServiceMsiApiVersion = "2019-08-01";
         private const string SecretHeaderName = "X-IDENTITY-HEADER";
 
-        private const string MsiEndpointInvalidUriError = "[Managed Identity] The environment variable IDENTITY_ENDPOINT contains an invalid Uri {0} in app service managed identity source.";
```

Comment:
Should this be in MsalErrorMessages?

Reply:
It is moved to MsalErrorMessage and removed from here.